doc/changes/changes_4.3.4.md

```diff
@@ -10,6 +10,12 @@ The release ignores vulnerability CVE-2023-7272 in transitive runtime dependency
 
 The release also pins Maven plugin versions to avoid plugin versions depending on the Maven version.
 
+The release also installs the Java versions required by Maven sources during the CI builds. Just set property `java.version` in your `pom.xml` and the required JDK will be installed in all GitHub workflows.
```

Comment:
How does this relate to the toolchains file?
Do the developers need to ensure a particular consistency?

Reply:
And don't we have particular workflows for java 11 as well as 17 (next java) to support a save transition path when java versions move on?
I am sorry, but maybe some words of introduction would be helpful here.
Maybe I just don't know the background?

Reply:
Proposal: Maybe we could add a (short) section to developer guide? 

Locally developers can install arbitrary JDK versions via the toolchain file (see ...). 
This remains unchanged. 

However, the CI build did install only JDK versions 11 and 17 and used two separate workflows, one for the mein build and running all tests and one for next-java using JDK version 17 and just verifying a limited set of checks.